keps/sig-cloud-provider/2436-controller-manager-leader-migration/README.md

```diff
@@ -349,18 +355,136 @@ feature without providing a configuration, the default configuration will reflec
 Yes. Once the feature is enabled via feature gate, it can be disabled by unsetting `--enable-leader-migration` on KCM
 and CCM.
 
+###### What happens if we reenable the feature if it was previously rolled back?
+
+This feature can be re-enabled without side effects.
+
 ###### Are there any tests for feature enablement/disablement?
 
 Yes. Unit & integration tests include flag/configuration parsing. E2E test will have cases with the feature enabled and
 disabled.
 
+### Rollout, Upgrade and Rollback Planning
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+The rollout may fail if the configuration file does not represent correct controller-to-manager.
+This can cause controllers referred in the configuration file to either be unavailable or run in multiple instances.
+
+The rollback may fail if the leader election of the controller manager is not properly configured.
+For example, multiple instances of the same controller manager are running without election, or none of the instances become the leader.
+In these situations, all controllers will be either unavailable or conflict among multiple instances.
```

Comment:
Same here, would be nice to have explicit examples of what configuration leads to each condition.